cloud/cmd/cloudcore/app/server.go

```diff
@@ -102,6 +103,10 @@ kubernetes controller which manages devices so that the device metadata/status d
 
 			config.CommonConfig.TunnelPort = *tunnelport
 
+			if changed := v1alpha1.AdjustCloudCoreConfig(config); changed {
+				updateCloudCoreConfigMap(config)
```

Comment:
also write back to local configure file? move this operation to line84, before ValidateCloudCoreConfiguration?

Reply:
the local configure file that mouted by the configmap will be updated automatically. and the autoadjust configuration is compliant with validation requirements